src/components/Popups/ReplyField/ItemList.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { SyntheticEvent } from 'react';
+import noop from 'lodash/noop';
+import DefaultItemRow from './DefaultItemRow';
+import { Collaborator } from '../../../@types';
+import './ItemList.scss';
+
+export type Props = {
+    activeItemIndex?: number;
+    filterString?: string;
+    itemRow?: JSX.Element;
+    items: Collaborator[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
+    setActiveItemIndex?: (index: number) => void;
+};
+
+const ItemList = ({
+    activeItemIndex = 0,
+    itemRow = <DefaultItemRow />,
+    items,
+    onSelect,
+    setActiveItemIndex = noop,
+    ...rest
+}: Props): JSX.Element => {
+    return (
+        <ul role="listbox" {...rest}>
+            {items.map((item, index) => {
+                if (!item) {
+                    return null;
+                }
+
+                return React.cloneElement(itemRow, {
+                    ...item,
+                    key: item.id,
+                    className: 'ba-ItemList-row',
+                    isActive: index === activeItemIndex,
```

Comment:
Unused in `DefaultItemRow`?

Reply:
> Unused in `DefaultItemRow`?

It is passed all the way down to `DatalistItem` imported from BUIE and used there.